### ISSUE TITLE
Add trashcan icon for even unfilled items

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#7352]
![X3](https://user-images.githubusercontent.com/1230959/74374049-a608d600-4e18-11ea-9b4c-93255ae4d123.jpg)
Here we observe
1. Unfilled item, Telephone
2. Half filled item, Wikipedia

which we started making, but say, now decided we don't want.

However, as they are still empty, there is no trashcan symbol to click!

Sure, "don't worry about it, they will go away, as they are empty."

But (Spaulding, a theoretical User In a Hurry, not me!) still wants them "Off the screen, and now!" But there is not even an X to click.

**Comment by nirmalmarian** [comment]
Hi! I would like to work on this issue. I am new to this so can someone kindly give me a head start?

**Comment by quincylvania** [comment]
> However, as they are still empty, there is no trashcan symbol to click!

@jidanni Yes, the trashcan symbol means that data will get deleted. Simply hiding the field wouldn't be deleting data, so that icon doesn't seem appropriate.

> But (Spaulding, a theoretical User In a Hurry, not me!) still wants them "Off the screen, and now!" But there is not even an X to click.

Is this behavior actually bothering you or another real person? We prioritize features that people actually want and need, not theoretical use cases.

---

> Hi! I would like to work on this issue. I am new to this so can someone kindly give me a head start?

@nirmalmarian Thanks for your interest, but we haven't decided to pursue this issue yet 😄

**Comment by jidanni** [comment]
> Is this behavior actually bothering you or another real person? 

Yes.

E.g., here on GitHub every Save button has a Cancel button next to it.
 
Otherwise even if a field is empty, the user worries he is about to save a
Capacity=
empty tag or something.

**Comment by tyrasd** [comment]
The only case where I could see this might be useful is when a preset has too many "unimportant" fields shown by default. But in that case, it would be better to improve the presets to only show the relevant fields. So, yeah… this is IMHO not worth the added clutter to the user interface.
